src/util/mask.js

```diff
@@ -90,14 +90,15 @@ const maskValidates = (mask, char) => {
  * @return {String}
  */
 export const maskText = (text, masked, dontFillMaskBlanks) => {
-  if (text == null) return ''
-  text = String(text)
-  if (!masked.length || !text.length) return text
+  if (!masked.length || !text) return text
```

Comment:
I just found out that `!'' === true`, so `!text.length` not only is redundant but will fail when `text === null`. Still learning :blush: